srcpkgs/AppStream/template

```diff
@@ -1,13 +1,13 @@
 # Template file for 'AppStream'
 pkgname=AppStream
 version=0.12.10
-revision=1
+revision=2
 wrksrc="appstream-${version}"
 build_style=meson
-configure_args="-Dgir=false -Dstemming=false -Dapidocs=false -Dqt=true"
+configure_args="-Dvapi=true -Dstemming=false -Dapidocs=false -Dqt=true"
```

Comment:
Can you use my commit from #20429 as-is? We want the build helper and build options set up.